src/index.html

```diff
@@ -7,9 +7,45 @@
 </head>
 <body>
 
+  <!--------------------------------------------------------------------------header -->
   <div id="printNavbar"></div>
+  <!------------------------------------------------------------------------- dashboard-cards -->
+  <div id="the-lounge" class="hide"></div>
 
-  
+
+  <!------------------------------------------------------------------------- footer -->
+  <div class="mt-5 pt-5 pb-5 footer">
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-5 col-xs-12 about-company">
+          <h2>Stay in contact</h2>
+          <p class="pr-5 text-white-50">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ac ante mollis quam tristique convallis </p>
+          <p><a href="#"><i class="fa fa-facebook-square mr-1"></i></a><a href="#"><i class="fa fa-linkedin-square"></i></a></p>
+        </div>
+        <div class="col-lg-3 col-xs-12 links">
```

Comment:
lets remove this links section for now.  to get these working will require a refactor.  if there is time we can circle back around